### PR TITLE
Damaging file system for cephfs-journal-tool

### PR DESCRIPTION
**Pull request by julpark-rh**
# Description
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-08D02U/

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_journal_mode.py [hunk 1]
```diff
@@ -65,6 +66,20 @@ def run(ceph_cluster, **kw):
             ",".join(mon_node_ips),
             extra_params=f",fs={fs_name}",
         )
+        # down all the osd nodes
+        mdss = ceph_cluster.get_ceph_objects("mds")
+        mds_list = []
+        for mds in mdss:
+            daemon_name = fs_util.deamon_op(mds, "mds", "stop")
+            mds_list.append((mds, daemon_name))
+        log.info("All the mds nodes are down")
```
Comment (hkadam134) [inline tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_journal_mode.py:75]: @julpark-rh  -- Can you also include "def check_deamon_status" to validate the expected daemon status after stopping ??

File: tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_journal_mode.py [hunk 2]
```diff
@@ -146,6 +161,14 @@ def run(ceph_cluster, **kw):
         log.error(traceback.format_exc())
         return 1
     finally:
+        # start mds
+        for mds in mds_list:
+            fs_util.deamon_op(mds[0], "mds", "reset-failed", service_name=mds[1])
+            fs_util.deamon_op(mds[0], "mds", "start", service_name=mds[1])
```
Comment (hkadam134) [inline tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_journal_mode.py:167]: @julpark-rh  -- Can you also include "def check_deamon_status" to validate the expected daemon status after starting ??

**Comment by neha-gangadhar** [comment]
@julpark-rh Please add logs for the execution.

File: tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_journal_mode.py (hunk 1 above)
Comment (julpark-rh) [reply tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_journal_mode.py:78]: added

File: tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_journal_mode.py (hunk 2 above)
Comment (julpark-rh) [reply tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_journal_mode.py]: added

**Review by hkadam134 — APPROVED** [review]
/lgtm

**Review by AmarnatReddy — APPROVED** [review]

**Pull request merged**